src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
+import { PrepearedPost } from '../../types/PrepearedPosts';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
 import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+interface Props {
+  post: PrepearedPost,
+}
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="post">
+    <h2 className="post__title" data-cy="post-title">
+      {post.title}
+    </h2>
+    <p className="post__body" data-cy="post-body">
+      {post.body}
+    </p>
+    <div className="post__comments comment">
+      { (post.comments)
+        ? <CommentList postComments={post.comments} />
+        : ''}
+    </div>
+    <div className="post__author">
+      {(post.user)
+        ? <UserInfo user={post.user} />
+        : ''}
+    </div>
```

Comment:
You shouldn't render this div if there is no user in post
{post.user && (
  <div> ... </div>
)}